### PR TITLE
Kluo/david merge

### PR DESCRIPTION
**Pull request by kluo**
@dcadams

### CODE REVIEW

**Comment by kluo** [comment]
retest this please

File: requirements/edx/github.txt [hunk 1]
```diff
@@ -43,10 +43,8 @@ git+https://github.com/edx/nose.git@99c2aff0ff51bf228bfa5482e97e612c97a23245#egg
 -e git+https://github.com/edx/event-tracking.git@0.2.0#egg=event-tracking
 -e git+https://github.com/edx-solutions/django-splash.git@7579d052afcf474ece1239153cffe1c89935bc4f#egg=django-splash
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
-
 -e git+https://github.com/Stanford-Online/edx-ora2.git@3039fd039e96c50b7ae4323fdeeb1bd0bc0ec2cd#egg=edx-ora2
 -e git+https://github.com/Stanford-Online/edx-submissions.git@d1beac80b846f0c0b0d956015755fc24d06adf25#egg=edx-submissions
-
```
Comment (dcadams) [inline requirements/edx/github.txt]: Can you remove this file from the PR as it's still a WIP.

**Comment by dcadams** [comment]
:+1:

**Comment by kluo** [comment]
retest this please

**Pull request merged**